Lib/unittest/test/testmock/testmock.py

```diff
@@ -226,6 +226,14 @@ class B(object):
             with self.assertRaisesRegex(InvalidSpecError,
                                         "Cannot spec attr 'B' as the spec_set "):
                 mock.patch.object(A, 'B', spec_set=A.B).start()
+            with self.assertRaisesRegex(InvalidSpecError,
+                                        "Cannot spec attr 'B' as the spec_set "):
+                mock.patch.object(A, 'B', spec_set=A.B).start()
```

Comment:
this stanza looks identical to the test immediately above it?  mis-paste or patch issue?

Reply:
yep messed up patch. fixed.